src/batVect.ml

```diff
@@ -1165,4 +1197,36 @@ struct
   let print ?(first="[|") ?(last="|]") ?(sep="; ") print_a out t =
     BatEnum.print ~first ~last ~sep print_a out (enum t)
 
+  module Labels =
+  struct
+    let init n ~f      = init n f
+    let concat ~v1 ~v2 = concat v1 v2
```

Comment:
That's true but sometimes you might want to flip the arguments around. Say you created a vector using a chain of pipe operations and want to prepend it to another one. The labels are optional. I'll remove it if you don't want it.